build/dev-env.sh

```diff
@@ -32,8 +32,6 @@ if [ -z "${SKIP_MINIKUBE_START}" ]; then
     test $(minikube status | grep Running | wc -l) -eq 2 && $(minikube status | grep -q 'Correctly Configured') || minikube start \
         --extra-config=kubelet.sync-frequency=1s \
         --extra-config=apiserver.authorization-mode=RBAC
-
-    eval $(minikube docker-env)
```

Comment:
I added this at https://github.com/kubernetes/ingress-nginx/pull/2795 because otherwise I can not bootstrap my local dev env.